com.unity.render-pipelines.universal/Documentation~/urp-shaders/urp-shaderlab-pass-tags.md

```diff
@@ -2,23 +2,37 @@
+| **UniversalGBuffer** | The Pass renders object geometry without evaluating any light contribution. Use this tag value in Passes that Unity must execute in the Deferred Rendering Path. |
+| **UniversalForwardOnly** | The Pass renders object geometry and evaluates all light contributions, similarly to when **LightMode** has the **UniversalForward** value. The difference from **UniversalForward** is that URP can use the Pass for both the Forward and the Deferred Rendering Paths.<br/>Use this value if a certain Pass must render objects with the Forward Rendering Path when URP is using the Deferred Rendering Path. For example, use this tag if URP renders a Scene using the Deferred Rendering Path and the Scene contains objects with shader data that does not fit the GBuffer, such as Clear Coat normals.<br/>If a shader must render in both the Forward and the  Deferred Rendering Paths, declare two Passes with the `UniversalForward` and `UniversalGBuffer` tag values.<br/>If a shader must render using the Forward Rendering Path regardless of the Rendering Path that the URP Renderer uses, declare only a Pass with the `LightMode` tag set to `UniversalForwardOnly`.<br/>If you use the SSAO Renderer Feature, add a Pass with the `LightMode` tag set to `DepthNormalsOnly`. For more information, see the `DepthNormalsOnly` value. |
+| **DepthNormalsOnly** | Use this value in combination with `UniversalForwardOnly` in the Deferred Rendering Path. This value lets Unity render the shader in the Depth and normal prepass. In the Deferred Rendering Path, if the Pass with the `DepthNormalsOnly` tag value is missing, Unity does not generate the ambient occlusion around the Mesh. |
 | **Universal2D** | The Pass renders objects and evaluates 2D light contributions. URP uses this tag value in the 2D Renderer. |
 | **ShadowCaster** | The Pass renders object depth from the perspective of lights into the Shadow map or a depth texture. |
 | **DepthOnly** | The Pass renders only depth information from the perspective of a Camera into a depth texture. |
```

Comment:
Minor: Maybe `DepthOnly` could appear just before in docs as `DepthNormalsOnly`?

Reply:
Thank you, I've added this to a separate task:
https://unity3d.atlassian.net/browse/DOC-3262